node/impl/full/sync.go

```diff
@@ -105,6 +106,26 @@ func (a *SyncAPI) SyncIncomingBlocks(ctx context.Context) (<-chan *types.BlockHe
 	return a.Syncer.IncomingBlocks(ctx)
 }
 
+func (a *SyncAPI) SlashFilterMinedBlock(ctx context.Context, bh *types.BlockHeader) (*types.BlockHeader, error) {
```

Comment:
Instead of adding this as an API endpoint, I think we can just create a separate SlashFilter in its own directory for the purpose of monitoring the entire network.